soc/espressif/common/Kconfig.amp

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2023 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_ENABLE_APPCPU
+	bool
+	default y
+	depends on (IPM || MBOX)
+	depends on SOC_SERIES_ESP32 || SOC_SERIES_ESP32S3
+	help
+	  This hidden configuration lets PROCPU core to map and start APPCPU whenever IPM is enabled.
+
+menu "Espressif AMP Config"
+
+config ESP32_APPCPU_IRAM_SIZE
+	hex "ESP32 APPCPU IRAM size"
+	default 0x10000
+	help
+	  Defines APPCPU IRAM area size in bytes.
+
+config ESP32_APPCPU_DRAM_SIZE
+	hex "ESP32 APPCPU DRAM size"
+	default 0x10000
+	help
+	  Defines APPCPU DRAM area size in bytes.
```

Comment:
these should be in dts

Reply:
That would be the next step I'm afraid, as it doesn't fit to the current model.